docs/install-overview.md

```diff
@@ -0,0 +1,130 @@
+OSG Site Installation Overview
+==============================
+
+This document outlines the overall installation process for an OSG site and provides many links into detailed installation, configuration, troubleshooting, and similar pages. If you do not see software-related technical documentation listed here, try the search bar to the left or contacting us at [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org).
+
+Plan the Site
+-------------
+
+If you have not done so already, [plan the overall architecture of your OSG site](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SitePlanning). It is recommended that your plan be sufficiently detailed to include the OSG hosts that are needed and the main software components for each host. Be sure to consider [the operating systems that OSG supports](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SupportedOperatingSystems). A simple way to organize this information is in a table; for example, a basic site might include:
+
+<span class="twiki-macro TABLE" sort="off"></span>
+
+| Purpose              | Host                                | Major Software                                           |
+|:---------------------|:------------------------------------|:---------------------------------------------------------|
+| Compute Element (CE) | `osg-ce.example.edu`                | OSG CE, HTCondor Central Manager, etc. (`osg-ce-condor`) |
+| Worker Nodes         | `w<em>NNN</em>.cluster.example.edu` | OSG worker node client (`osg-wn-client`)                 |
+
+Prepare the Batch System
+------------------------
+
+The assumption is that you have an existing batch system at your site. Currently, we support [HTCondor](http://research.cs.wisc.edu/htcondor/), [LSF](http://www-03.ibm.com/systems/platformcomputing/products/lsf/), [PBS](http://www.pbsworks.com) and [TORQUE](http://www.adaptivecomputing.com/products/open-source/torque/), [SGE](http://en.wikipedia.org/wiki/Oracle_Grid_Engine), and [Slurm](http://slurm.schedmd.com) batch systems.
+
+The easiest way to add a site to OSG is to install the OSG Compute Element (CE) on the central host of your batch system. In any case, before proceeding be sure that you can submit and successfully run a job from your OSG CE host into your batch system.
+
+Add OSG Software
+----------------
+
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+
+<span class="twiki-macro NOTE"></span> For sites with more than a trivial number of hosts, it is recommended to use some sort of configuration management tool to install, configure, and maintain your site. While beyond the scope of OSG’s documentation to explain how to select and use such a system, some popular configuration management tools are [Puppet](http://puppetlabs.com), [Chef](https://www.chef.io), [Ansible](https://www.ansible.com), and [CFEngine](http://cfengine.com).
```

Comment:
What is this NOTE?  I think we can have:

    !!! note
        For sites with more than a trivial...

This is the extension: https://pythonhosted.org/Markdown/extensions/admonition.html